### ISSUE TITLE
[UI] Default sort order for Flows view

### ISSUE THREAD

**Issue by kriko** [kestra-io/kestra#3112]
### Feature description

Currently the default sort order for flows view is to sort by flow id, ascending.
I propose to enable two layer sorting - first sort by namespace name, asc then sort by flow id.

Better visibility when viewing flows across all namespaces or a parent namespace with child namespaces.

**Comment by anna-geller** [comment]
you're right, we plan to add a [Saved search](https://github.com/kestra-io/kestra/issues/1397), allowing to define which fields you would like to see and the sort order and save as view/custom dashboard/saved search

**Comment by SyedMDot** [comment]
@anna-geller please assign this issue to me

**Comment by SyedMDot** [comment]
![image](https://github.com/user-attachments/assets/daca4a88-22ea-4c28-88b5-252a4b3dd6d1)

Has the task been completed? I see that sorting is now enabled for both namespace and flow ID.
OR
Am I misunderstanding the question?

**Comment by anna-geller** [comment]
the sorting by more than one column still doesn't work so the issue is open and you can contribute 👍 I recorded a quick demo to show what is meant exactly: https://share.descript.com/view/agWkLFKLeI0

**Comment by SyedMDot** [comment]
Okay, I got the point. I'll try my best to sort it out.

**Comment by VinaySatrasala** [comment]
> Okay, I got the point. I'll try my best to sort it out.

Hello @SyedMDot have u started working on this issue

**Comment by SyedMDot** [comment]
Hello I've completed my work on this issue, @anna-geller could you please validate from the below video, is the solution is as expected?

https://drive.google.com/file/d/15SXMRqopQpO_cRl3R9MWiPT0N9LKZzDW/view?usp=sharing

**Comment by SyedMDot** [comment]
> Hello I've completed my work on this issue, @anna-geller could you please validate from the below video, is the solution is as expected?
> 
> https://drive.google.com/file/d/15SXMRqopQpO_cRl3R9MWiPT0N9LKZzDW/view?usp=sharing

After your confirmation, I'll create a PR.

Thank you!

**Comment by Ben8t** [comment]
Thanks @SyedMDot for the video! I think it's not 100% right: at 0.15 second on your video we can see that namespace is sorted (blueprint on top) but then you click on id sort and the namepsace change (to company.team). In this case the namespace sort should keep the same order.

**Comment by SyedMDot** [comment]
> Thanks @SyedMDot for the video! I think it's not 100% right: at 0.15 second on your video we can see that namespace is sorted (blueprint on top) but then you click on id sort and the namepsace change (to company.team). In this case the namespace sort should keep the same order.

Hi @Ben8t , in the video, we sort the ID first in asc order, then namespace in asc order, and same for desc order, after then both are sorted as expected.

Means first sort by ID, then by namespace, in both ascending and descending order.

**Comment by anna-geller** [comment]
This might be a nice issue for first-time contributors

**Comment by manuj-chadha** [comment]
Hey, I would be happy to contribute to this issue! Can you assign it to me?

**Comment by MilosPaunovic** [comment]
Hi @manuj-chadha,  

You're already assigned to another issue in our repository, and we usually assign only one at a time to new contributors. Let's make sure to wrap that one up before jumping onto another.  🚀

**Comment by balayokesh** [comment]
Can i work on this issue

**Comment by MilosPaunovic** [comment]
Hey @balayokesh, yes, yes you can, thanks! 🚀

**Comment by MilosPaunovic** [comment]
Hey @balayokesh, can we expect a new PR soon?

**Comment by balayokesh** [comment]
> Hey @balayokesh, can we expect a new PR soon?


Yes sure.  I'll be opening a new PR within the next 10 hours.
